### PR TITLE
Update prepare for battle

### LINKED ISSUE #1456: [FEATURE] Update Prepare for battle screens
## 🚀 Feature

- Update the assets with the new color palette.

- Update the portraits.

### Assets

https://drive.google.com/drive/folders/1ra-sO0G9YwVHueII5B7PgJm8Xqcua6xz?usp=drive_link

### References
reference
reference 2

### PR DESCRIPTION
**Pull request by BertovDev**
Closes #1456 

## Motivation

Update prepare for battle screen

## Summary of changes

List changes in code.

## How has this been tested?

Describe the steps you performed to test this PR.

## Test additions / changes

List tests added/updated.

## Checklist
- [ ] I have tested the changes locally.
- [ ] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Comment by jazimp** [comment]
I reviewed the visual implementation and it's looking good!

File: client/Assets/Images/UI/BattleUI/PrepareForBattle/New/1(1).png
Comment (AngieDutra) [inline client/Assets/Images/UI/BattleUI/PrepareForBattle/New/1(1).png:1]: Try to avoid naming the images with numbers

**Review by AngieDutra — CHANGES_REQUESTED** [review]
I noticed the UI label was misplaced. However neither the reference or implementation works for all images without covering the character's face. I talked with @jazimp and she said the label should be placed in the character's name container anchored in the middle right side of it.

This PR implementation:
Captura de pantalla 2024-02-26 a las 11 38 44 a  m

The reference:
306440100-ce401fef-b1cf-4e56-aa52-9b1e61b59873

File: client/Assets/Images/UI/BattleUI/PrepareForBattle/New/1(1).png
Comment (AngieDutra) [reply client/Assets/Images/UI/BattleUI/PrepareForBattle/New/1(1).png:1]: Also, please review all the new images max sizes.
Captura de pantalla 2024-02-26 a las 11 47 17 a  m

**Comment by AngieDutra** [comment]
The team requested a las minute fix: the current text "Be the last standing" should be -> "Be the last one standing". Checked with @BertovDev and he will add this here. I'll wait for that commit to review this PR

**Review by AngieDutra — APPROVED** [review]
LGTM
✔️ Text was changed
✔️ Valtimer's victory animation was fixed (thought it was going to be fixed [here](https://github.com/lambdaclass/curse_of_mirra/pull/1496) but it works"
✔️ Each character's icon in prepare screen is correct
✔️ The you label is fixed
✔️ Files were correctly added in the heriarchy
✔️ Sprite sizes were fixed
✔️ Tested with more players and the label is correctly placed
Just a note: I believe the "you" label is not working visually anymore so in a near future we should rethink how we are highlighting the player's own character. Already talked with @jazimp about this. Created issue #1517 regarding this

**Review by AngieDutra — APPROVED** [review]

**Comment by agustinesco** [comment]
> LGTM ✔️ Text was changed ✔️ Valtimer's victory animation was fixed (thought it was going to be fixed [here](https://github.com/lambdaclass/curse_of_mirra/pull/1496) but it works) ✔️ Each character's icon in prepare screen is correct ✔️ The you label is fixed ✔️ Files were correctly added in the heriarchy ✔️ Sprite sizes were fixed ✔️ Tested with more players and the label is correctly placed Just a note: I believe the "you" label is not working visually anymore so in a near future we should rethink how we are highlighting the player's own character. Already talked with @jazimp about this. Created issue #1517 regarding this

I think the "You" label is a path while we implement the username and user registration system, when we implement this the user will be able to recognize himself with his username right?

**Review by agustinesco — CHANGES_REQUESTED** [review]
Is there any reason to add the new images to a folder called "New"? we think the images should just be on the "PrepareForBattle" Folder

Also lets rename the images to delete the number, for example:
`client/Assets/Images/UI/BattleUI/PrepareForBattle/New/H4ck(1).png`
Should be 
`client/Assets/Images/UI/BattleUI/PrepareForBattle/New/H4ck.png`

**Comment by agustinesco** [comment]
Tested the prepare for battle animation and is working correctly with every character both in the unity editor and in an android build! should be ready to merge when we fix the files names and folders

**Comment by AngieDutra** [comment]
Good one! While you are at this maybe you could delete the "new" folder and move the images inside that to the parent folder, according to our desired heriarchy

**Review by agustinesco — APPROVED** [review]
🪨

**Review by AngieDutra — APPROVED** [review]
✔️ Did a final asset check and works just fine

**Pull request merged**